catalog/opensearch/catalog-opensearch-source/src/test/java/org/codice/ddf/opensearch/source/OpenSearchSourceTest.java

```diff
@@ -901,7 +900,12 @@ protected InputTransformer lookupTransformerReference(String namespaceUri)
     @Override
     protected SecureCxfClientFactory createClientFactory(
         String url, String username, String password) {
-      return this.factory;
+      return factory;
+    }
+
+    @Override
+    protected SecureCxfClientFactory getClientFactory() {
+      return factory;
     }
```

Comment:
❓ Why do you need to override the `getClientFactory` method here? Can you get away with just overriding `createClientFactory`?

Reply:
OpenSearchSource uses two different paths for acquiring a SecureCxfClientFactory instance. One uses the instance member `factory` and the other calls `createClientFactory`. I didn't look close enough to see why it was written that way, but these changes should not affect the original behavior.

Reply:
You are right- thanks for the comment!